lib/slack/renderer/push.js

```diff
@@ -13,24 +13,32 @@ class Push extends Message {
     this.push = push;
   }
 
+  get commits() {
+    return this.push.commits.map((commit) => {
+      const message = commit.message.split('\n')[0];
+      const sha = commit.id.substring(0, 8);
```

Comment:
does the `sha` really add value to the overall message? It seems like noise, so I would leave it off 😬

Reply:
tried it out: It looks super weird without the sha